### DISCUSSION TITLE
Readme showing on domain file instead of index on github pages

### DISCUSSION THREAD

**Discussion by rylanwill** [statamic/cms#4559] [Troubleshooting] [answered]
Hey guys,

I'm trying to launch a statamic starter site on a domain I just purchased on github pages as a test, but I'm only seeing the readme file when I visit the domain. 

I've done a bit of googling but haven't found anything specific to this. Any help would be appreciated!

Repo:
https://github.com/rylanwill/mazzas

URL: http://mazzas.com.au/

Pages setup: 
![image](https://user-images.githubusercontent.com/88601366/138521836-c7495fa1-f7f7-4934-8cea-1dc15e948970.png)

**Comment by edalzell** [comment, marked as answer]
Not really a Statamic thing, but GH pages can only do static sites right? if so, you'll need to use the SSG to generate your site then deploy that.

**Reply by jasonvarga** [reply]
That's right. GitHub pages is a feature that lets you host a Jekyll site. It's not used to deploy Statamic sites. But visiting the site you linked to, it's showing the Statamic starter kit home page, so you must have worked it out 👍

**Reply by rylanwill** [reply]
Yep, all sorted - turns out gh pages wasn't the best way to go. Appreciate the response guys!
